core/src/oracle_state.rs

```diff
@@ -258,13 +258,18 @@ impl<'a> OraclePool<'a> {
                 scans.push(local_scan);
             }
 
+            let network_prefix =
```

Comment:
It was crucial that `OracleConfig::oracle_address` is encoded as it allows us to obtain the network prefix used to generate `ballot_token_owner_address_str`, since we no longer have `OracleConfig::on_mainnet`.